ui/test/nuxeo-favorites-toggle-button.test.js

```diff
@@ -48,12 +48,25 @@ suite('nuxeo-favorites-toggle-button', () => {
       ]);
     });
 
+    teardown(() => {
+      if (window.confirm.restore) {
+        window.confirm.restore();
+      }
+    });
+
     test('it should display the document as favorite', () => {
       expect(element.favorite).to.be.true;
     });
 
-    test('toggle should remove document from favorites', async () => {
+    test('it should display a confirm popup', () => {
+      const confirmSpy = sinon.stub(window, 'confirm');
+      tap(element);
+      expect(confirmSpy.called).to.be.true;
+    });
+
+    test('toggle should remove document from favorites when clicked ok', async () => {
```

Comment:
We should also test what happens if the user clicks cancel in this situation. We need to make sure that nothing happens.

Reply:
I would say to add a new test case for `toggle should not remove document from favorites when clicked cancel`